### PR TITLE
Add RxJS

### PR DESCRIPTION
**Pull request by filipesilva**
The current RxJS documentation site is https://rxjs.dev/.

It is very similar to Angular documentation site (https://angular.io/) so I reused most code.

Images on the documentation site seem to be broken and so the scrapper cannot download them. You can see an example of a broken image at https://rxjs.dev/api/operators/buffer.

Related to https://github.com/freeCodeCamp/devdocs/issues/939

cc @benlesh

### CODE REVIEW

File: lib/docs/scrapers/rxjs.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    self.links = {
+      home: 'https://rxjs-dev.firebaseapp.com/',
+      code: 'https://github.com/ReactiveX/rxjs'
+    }
+
+    options[:max_image_size] = 256_000
+
+    options[:attribution] = <<-HTML
+      &copy; 2010&ndash;2018 Google, Inc.<br>
+      Licensed under the Creative Commons Attribution License 4.0.
```
Comment (filipesilva) [inline lib/docs/scrapers/rxjs.rb]: @benlesh can you point me towards the right attribution for RxJS please?

File: lib/docs/scrapers/rxjs.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    self.links = {
+      home: 'https://rxjs-dev.firebaseapp.com/',
+      code: 'https://github.com/ReactiveX/rxjs'
+    }
+
+    options[:max_image_size] = 256_000
+
+    options[:attribution] = <<-HTML
+      &copy; 2015&ndash;The RxJS Project Developers<br>
+      Code licensed under an Apache-2.0 License. Documentation licensed under CC BY 4.0.
```
Comment (filipesilva) [inline lib/docs/scrapers/rxjs.rb]: @benlesh can you confirm if this attribution is correct?

I took the year from the first RxJS release (https://github.com/ReactiveX/rxjs/releases/tag/0.0.0-prealpha.1) and the license text from https://rxjs-dev.firebaseapp.com/.

**Comment by filipesilva** [comment]
@benlesh done.

**Comment by barberousse** [comment]
Any timeline on seeing this in production?

**Comment by jmerle** [comment]
Unfortunately not. Although all maintainers now have push access to the GitHub repository, we can't deploy new documentation yet because that process is a little more complicated involving more than just a simple merge. @QuincyLarson came up with the great idea to have a meeting with all maintainers in which @Thibaut can show how deploying works, but that meeting hasn't been planned yet (I am waiting for Quincy to make a move on this).

After all maintainers are capable of deploying new changes (after that meeting, hopefully) we should be able to get through the PR's relatively quickly.

**Comment by QuincyLarson** [comment]
@barberousse Thanks for your patience. As @jmerle said, we're getting close to shipping some critical-path code on the freeCodeCamp repo and then we'll gear up for deploying the new DevDocs code.

**Comment by barberousse** [comment]
@jmerle @QuincyLarson Thanks for getting back to me on that

**Comment by samosaara** [comment]
Heya, just found about this tool and been loving it so far, sadly it doesn't have RXJS 6 docs on it, and it has been 3 months since last update... Any news?

**Comment by jmerle** [comment]
Thanks! Deployed to [https://devdocs.io/rxjs/](https://devdocs.io/rxjs/).

**Comment by filipesilva** [comment]
Hooray! Thank you so much :D

**Pull request merged**